src/point_cloud_processor.h

```diff
@@ -82,7 +98,8 @@ class PointCloudProcessor {
         for (int i = 0; i < static_cast<int>(pc_msgs.size()); ++i) {
             auto range_ch = static_cast<sensor::ChanField>(sensor::ChanField::RANGE + i);
             auto range = lidar_scan.field<uint32_t>(range_ch);
-            ouster::cartesianT(points, range, lut_direction, lut_offset,
+            auto range_masked = mask.rows() == range.rows() && mask.cols() == range.cols() ? range * mask : range;
```

Comment:
Should there be a way to let the user know they specified an incorrectly-sized image? Otherwise, they might be confused as to why the mask they have is ineffective.

Reply:
Added a condition that checks if mask match the sensor resolution